components/cam/src/physics/cam/shoc_intr.F90

```diff
@@ -303,7 +303,7 @@ subroutine shoc_init_e3sm(pbuf2d, dp1_in)
     dp_frac_idx = pbuf_get_index('DP_FRAC')     ! Deep convection cloud fraction
     icwmrdp_idx = pbuf_get_index('ICWMRDP')     ! In-cloud deep convective mixing ratio
     sh_frac_idx = pbuf_get_index('SH_FRAC')     ! Shallow convection cloud fraction
-    relvar_idx  = pbuf_get_index('RELVAR')      ! Relative cloud water variance
+    qc_relvar_idx  = pbuf_get_index('QC_RELVAR')      ! Relative cloud water variance
```

Comment:
Again, here, will the change in pbuf variable name impact SHOC compatibility with MG2?  Does MG2 use the "qc_relvar" name or does it stick with "relvar"

Reply:
@AaronDonahue 
Thanks for the good review! 
I'm checking that for the CLUBB/MG2 end right now and will update that along with other issues in a followup commit (thought about that, but haven't done anything with it ...).